src/main/java/racing/RacingGame.java

```diff
@@ -0,0 +1,41 @@
+package racing;
+
+
+import racing.strategy.MoveStrategy;
+import racing.strategy.RacingMoveStategy;
+import racing.view.InputView;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class RacingGame {
+
+    private MoveStrategy moveStrategy;
+
+    public static void main(String[] args) {
+        int countOfRacingCar = InputView.getRacingCarInput();
+        int countOfRacing = InputView.getRacingInput();
+
+        MoveStrategy moveStategy = new RacingMoveStategy();
+        RacingGame racingGame = new RacingGame(moveStategy);
+        List<RacingCar> racingCarList = racingGame.setRacing(countOfRacing);
+        List<RacingCar> racingResult = racingGame.startRacing(countOfRacing, racingCarList);
+        System.out.println(racingResult);
+        ResultView.viewRacingResult(racingResult);
+    }
+    public RacingGame() {}
+
+    public RacingGame(MoveStrategy moveStrategy) {
+        this.moveStrategy = moveStrategy;
+    }
+
+    public List<RacingCar> setRacing(int countOfRacingCar) {
+       return Stream.generate(() -> new RacingCar(moveStrategy)).limit(countOfRacingCar).collect(Collectors.toList());
```

Comment:
```suggestion
       return Stream.generate(() -> new RacingCar(moveStrategy))
                   .limit(countOfRacingCar)
                   .collect(Collectors.toList());
```

스트림을 사용할 땐 한 줄에 하나의 메소드만 사용해주세요!